src/Frontend/Views/GameWindowLayout.tsx

```diff
@@ -210,7 +213,8 @@ export function GameWindowLayout({
           <HoverPlanetPane />
           <ArtifactHoverPane />
 
-          <TutorialPane tutorialHook={tutorialVisible || isTutorialWorld} />
+          {showTutorial && <TutorialPane />}
```

Comment:
what happens if you are spectating and doing the tutorial? maybe we should block spectator mode if the tutorial config is selected?

Reply:
Done. Removed game mode selection menu for tutorial.